### DISCUSSION TITLE
How can I disable pokemon on terminal startup

### DISCUSSION THREAD

**Discussion by mateus-sartorio** [JaKooLit/Fedora-Hyprland#113] [Q&A] [answered]
The random pokemon that appears on terminal startup is really cool, but I use my computer for work, and I don't want him seeing a pokemon every time I open the terminal to show him something.

**Comment by JaKooLit** [comment, marked as answer]
https://github.com/JaKooLit/Hyprland-Dots/wiki/FAQ#%EF%B8%8F-dont-like-pokemon-colorscripts

but on the installation script, you were given the option wether to install or not the pokemon?
